<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganPrintExportPanelControls</class>
 <widget class="QWidget" name="OrganPrintExportPanelControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Awesome View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportSTLButton">
     <property name="toolTip">
      <string>Export the segmentation into a 3D printable format.</string>
     </property>
     <property name="text">
      <string>Export in STL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="toolTip">
      <string>Save the project and open later if you desires</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Selection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="selectedNodeLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>No selection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="slicerButton">
     <property name="text">
      <string>Open in 3D Slicer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="smoothingCheckBox">
     <property name="text">
      <string>Smoothen before opening</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
